### DISCUSSION TITLE
Application built with cursive opens its own window on Windows

### DISCUSSION THREAD

**Discussion by jasperweiss** [gyscos/cursive#593] [Q&A]
Hi! I came across a [Spotify client called ncspot](https://github.com/hrkfdn/ncspot) which uses cursive.
One thing I noticed is that on Windows, it doesn't run within my current Terminal app but opens a new window instead. Why is that? is there a way to make it run within the current terminal as it does on MacOS?

![image](https://user-images.githubusercontent.com/29832921/127221810-625ff973-7ed3-47c7-963b-a78a627e470d.png)

**Comment by gyscos** [comment]
Hi!

I think ncspot uses the [pancurses](https://github.com/ihalila/pancurses) backend by default.

If you try one of the pancurses examples, does it have the same behavior?

**Reply by jasperweiss** [reply]
Hi, I just tried the most recent build and it works properly now! Not sure what changed though
